src/events/adapters/importers/fit/importer.fit.ts

```diff
@@ -352,7 +353,9 @@ export class EventImporterFIT {
       startDate,
       endDate,
       this.getActivityTypeFromSessionObject(sessionObject),
-      this.getCreatorFromFitDataObject(fitDataObject)
+      this.getCreatorFromFitDataObject(fitDataObject),
+      '',
+      fitDataObject.file_ids[0].manufacturer
```

Comment:
What do you need this info for?

Reply:
Just to check what mapping logic can be applied to fit file. I am yet to check how Strava differs in case of `intensity` field. I know, only Garmin is supported for activity import right now, but in case it's added some time in the future, then it might come in handy.